src/synthesized_datasets/_datasets.py

```diff
@@ -51,6 +55,14 @@ def load(self) -> _pd.DataFrame:
         df.attrs["name"] = self.name
         return df
 
+    def load_spark(self, spark: _ps.SparkSession) -> _ps.DataFrame:
```

Comment:
What are your thoughts on making spark an optional kwarg in `load`? When passed in it returns a spark dataframe, otherwise a pandas one is returned.